### PR TITLE
Added the main video and the sidebar of suggested videos to be played

### PR DESCRIPTION
**Pull request by descholar-ceo**
### What does this PR do?
It adds a main player page to the project

### Changes we made in this PR
- [x] Created the skeleton HTML of the player page
- [x] Added icons in the navbar and on the bottom of the video player section
- [x] Add the thumbnails to the sidebar
- [x] Styled everything with the CSS

### CODE REVIEW

**Comment by santiago-dev96** [comment]
# Pull Request under review by @santiago-rodrig (TSE)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+<head>
+    <meta charset="UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <link rel="stylesheet" type="text/css" href="./index.css"/>
+    <title>YouTube video player page</title>
+</head>
+<body>
+    <nav>
+        <ul class="nav-left-section">
+            <li><img src="/img/fontawesome/align-justify.svg"></li>
```
Comment (santiago-dev96) [inline index.html]: You should use relative paths, that way you'll make sure that images' path is being properly specified.
```suggestion
            <li><img src="./img/fontawesome/align-justify.svg"></li>
```

**Review by santiago-dev96 — CHANGES_REQUESTED** [review]
# Changes required 🚫

Your file `index.html` is not properly displaying images, please fix that.

Screen Shot 2020-06-03 at 9 31 00 AM

Screen Shot 2020-06-03 at 9 31 17 AM

Kindly submit another CRR (Code Review Request) after implementing the required changes.

You can find me in slack as **srodrig**, happy coding! :wave:

## P.D.

Please try to make your PR's title and description more meaningful, think about yourselves as professional developers, the PR summary should thoroughly describe the changes being introduced by the PR, and the title should label the major change that is being made.

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Change Required :recycle: 

### Hello @descholar-ceo and @Haywhizzz    :handshake: 

### Good job so far :clap:  your project is looking good :medal_sports: 

### there are some changes to get the project approved :point_down:
- You project readme is empty, the project should contain a descriptive readme:

- [x] The project has a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- please read again this https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/ to understand what is professional github repository.
- The index.html didn't passe the validator:
Screenshot from 2020-06-04 00-26-31
Screenshot from 2020-06-04 00-26-42
Screenshot from 2020-06-04 00-26-52
- [x] Check HTML markup with this validator https://validator.w3.org/#validate_by_upload. (Students were asked to use this one, so in most cases their code should be already validated) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/validation_requirement.png) from Pathwright)
- The share should be a button:
- [x] It has a share button ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_share_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))
### Please implement all these required changes and ask for another code review.

**Review by HADDADSOHAIB — APPROVED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Project Approved :+1: 

### Hello @descholar-ceo and @Haywhizzz    :handshake: 

### Good job :clap:  you did all the required changes :medal_sports: 

### This project is approved you can go on and merge it.

File: index.html (hunk 1 above)
Comment (descholar-ceo) [reply index.html]: Fixed this

**Pull request merged**